### PR TITLE
v3 reCaptcha validation form

### PR DESCRIPTION
**Pull request by vaibhavAP**

### CODE REVIEW

File: support.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	<link rel="stylesheet" href="css/responsive.css">
 	<link rel="stylesheet" href="js/sweetalert2//dist/sweetalert2.min.css">
 	<script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
-	<script src='https://www.google.com/recaptcha/api.js'></script>
+	<script src="https://www.google.com/recaptcha/api.js?render=6LeYMbcZAAAAAGW_wktwCx5gZe1qDnkLFNNij2GI"></script>
```
Comment (misterhackney) [inline support.html:44]: Why are you not using the provided site_key in your Pull Requests?
It includes localhost as an option, so no need to use anything different even during your testing.

File: js/custom.support.validate.js [hunk 2]
```diff
@@ -123,38 +123,57 @@ function addError(formGroup, error) {
...
+                if(res.status === 200){
+                    swal.fire({
+                        title: "Thank You!",
+                        type: "success",
+                        confirmButtonText: 'Ok'
+                    });
+                    resetForm();
+                } else if(false){  //condition to be used when response from server is returned
+                    swal.fire({
+                        title: "You are Suspicious User!",
```
Comment (misterhackney) [inline js/custom.support.validate.js:159]: Let's use a more generic & nice phrase here. Like "Your submission failed. Please try again."

File: js/custom.support.validate.js [hunk 3]
```diff
@@ -154,9 +154,9 @@ function showSuccess() {
                         confirmButtonText: 'Ok'
                     });
                     resetForm();
-                } else if(false){  //condition to be used when response from server is returned
+                } else if(false){  //condition to be filled according to the returned response from server for suspicious user
```
Comment (misterhackney) [inline js/custom.support.validate.js:157]: Do not put the words "Suspicious user" anywhere in our code. That is not appropriate. We already asked this below.

Also, you do not know why a non-successful response is given. There could be alot of reasons we don't get a response == 200.  So, just use a generic response. Generic comment.... and stop using the word "suspicious"

Comment (vaibhavAP) [reply js/custom.support.validate.js]: Do you want to show same error in alert when recaptcha failed and when some error occurred at the server or you will show different messaage?

Comment (misterhackney) [reply js/custom.support.validate.js]: Recaptcha doesn't "fail" so your question makes no sense.
Recaptcha just delivers a score.  If score isn't above a specific threshold, we will refuse to allow interaction with the back-end server to reduce spam load.
So, study recaptcha v3 deeper so you understand this "score" concept or reword your question so it aligns with the terminology found in Recaptcha v3.

**Pull request merged**